console/services/backup_service.py

```diff
@@ -157,7 +157,7 @@ def delete_group_backup_by_backup_id(self, tenant, region, backup_id):
         if not backup_record:
             raise ErrBackupRecordNotFound
         if backup_record.status == "starting":
-            return ErrBackupInProgress
+            return backup_record_repo.delete_record_by_backup_id(tenant.tenant_id, backup_id)
```

Comment:
When a backup is in progress and needs to be deleted, only the console record is deleted and the task of the data center is ignored. Because the backup is not complete, it will be an error to delete the data center records

Reply:
Not reasonable, perhaps the cluster side should also be changed. the method `check_unfinished_backup ` depends on cluster-side data.